sdk/src/types/block/core/mod.rs

```diff
@@ -21,8 +21,8 @@ pub use self::{
     validation::{ValidationBlock, ValidationBlockBuilder},
     wrapper::{BlockHeader, BlockWrapper},
 };
+use super::parent::Parents;
```

Comment:
Not a big fan of mixing super and crate tbh, I don't see the logic here, why parents and not protocol? It's inconsistent with the rest of the codebase, I'd keep it in the crate import.

Reply:
fixed